### PR TITLE
[UIPO-70] Carousel: fix responsive design and console warning

### PR DESCRIPTION
**Pull request by dimas0211**

### CODE REVIEW

File: src/components/Carousel/Carousel.js [hunk 1]
```diff
@@ -141,10 +141,11 @@ class Carousel extends Component {
     const { children } = this.props;
     const { widthOfSlide } = this.state;
     const style = { maxWidth: widthOfSlide };
+    // console.log(children);
```
Comment (nadiiaLebedenko) [inline src/components/Carousel/Carousel.js]: Remove this comment, please

File: src/components/Carousel/Carousel.spec.js [hunk 2]
```diff
@@ -8,13 +8,13 @@ describe('<Carousel />', () => {
...
   beforeEach( () => {
     items = [
-      <div id={1}><div key={1}>hello1 </div></div>,
-      <div id={2}><div key={2}>hello 2</div></div>,
-      <div id={3}><div key={3}>hello3 </div></div>,
-      <div id={4}><div key={4}>hello 4</div></div>,
-      <div id={5}><div key={5}>hello 5</div></div>,
-      <div id={6}><div key={6}>hello 6</div></div>,
-      <div id={7}><div key={7}>hello 7</div></div>,
+      <div key={1}><div key={1}>hello1 </div></div>,
```
Comment (nadiiaLebedenko) [inline src/components/Carousel/Carousel.spec.js]: It is a bad practice to have different elements with same key. Plus keys should be meaningfull

**Review by nadiiaLebedenko — CHANGES_REQUESTED** [review]

File: src/components/Carousel/Carousel.spec.js (hunk 2 above)
Comment (dimas0211) [reply src/components/Carousel/Carousel.spec.js]: done)

File: src/components/Carousel/Carousel.js (hunk 1 above)
Comment (dimas0211) [reply src/components/Carousel/Carousel.js]: done)

**Review by nadiiaLebedenko — APPROVED** [review]

**Review by Nazar-bnm — APPROVED** [review]

**Pull request merged**